nengo_dl/tests/test_graph_optimizer.py

```diff
@@ -1111,12 +1111,26 @@ def test_remove_reset_incs():
 
     # convinc converted to convset
     x = dummies.Signal()
-    operators = [Reset(x), ConvInc(dummies.Signal(), dummies.Signal(), x, None)]
+    conv = nengo.Convolution(1, input_shape=(3, 3, 1))
+    operators = [Reset(x), ConvInc(dummies.Signal(), dummies.Signal(), x, conv)]
     new_operators = remove_reset_incs(operators)
     assert len(new_operators) == 1
     assert isinstance(new_operators[0], transform_builders.ConvSet)
     assert new_operators[0].Y is x
 
+    # convtransposeinc converted to convtransposeset
+    if not compat.is_dummy_type(compat.ConvTransposeInc):
```

Comment:
Rather than checking `is_dummy_type` here, I would check the `nengo` version. It just makes it easier when we increase the minimum nengo version to find the places in the code where we can remove backwards compatibility logic.

Reply:
Makes sense to me :+1: